shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/controler/DatasetApiController.java

```diff
@@ -657,7 +667,6 @@ public void massiveDownload(String format, List<Dataset> datasets, HttpServletRe
 	}
 
 	/**
-<<<<<<< HEAD
```

Comment:
Hi Jean-Come, could you please remove the head comment? Thank you and with kind regards, Michael